scania-digital-design-system/sdds
# fix(banner): changed the padding/margin of the close-btn and icon to …

changed the padding/margin of the close-btn and icon to align with design



**Describe pull-request**  
Changed the padding and margin of the prefix icon and close button on the banner to align with the design.

**Solving issue**  
Fixes: [AB#2319](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2319)

**How to test**  
1. Go to the storybook page below.
2. Check in the banner component.
3. Compare the position of the prefix icon and close button to the design. 

**Screenshots**  
![Screenshot 2022-10-05 at 14 08 53](https://user-images.githubusercontent.com/62651103/194056969-1f763050-bd2f-44c9-b821-d5abbb954472.png)


**Additional context**  
-
